stylelint.config.js

```diff
@@ -25,6 +24,15 @@ module.exports = {
     'scss/at-extend-no-missing-placeholder': null, // Fontawesome does not provide placeholders for individual icons
     'scss/at-import-no-partial-leading-underscore': null,
     'scss/at-rule-conditional-no-parentheses': null,
+    'scss/at-rule-no-unknown': [
+      true,
+      {
+        'ignoreAtRules': [
+          'extends',
+          'tailwind'
+        ]
+      }
+    ],
```

Comment:
May You add a comment here, why thats neccessary?

Reply:
sure! cd2ec5c5d019fda155f1f0c368c09404f3983798